### PR TITLE
[CircleGraph] Introduce formatted index.js

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce index.js with format applied
- added '// eslint-disable-next-line' for variables don't follow camelCase
- changed '==' to '==='
- use tab size 4 to follow original source

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #520

**Review by hyunsik-yoon — DISMISSED** [review]

File: media/CircleGraph/index.js [hunk 1]
```diff
@@ -0,0 +1,1142 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
```
Comment (llFreetimell) [inline media/CircleGraph/index.js:39]: Could you add which source code is referenced? :)

Comment (seanshpark) [reply media/CircleGraph/index.js:39]: This source is same as `external/index.js` but applied format and fixes for lint tools.

Comment (seanshpark) [reply media/CircleGraph/index.js:39]: added comment

**Review by hyunsik-yoon — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]

**Pull request merged**